velo-impl/src/main/java/me/aurium/beetle/velocity/VelocityCommandData.java

```diff
@@ -1,19 +1,14 @@
 package me.aurium.beetle.velocity;
 
 import com.velocitypowered.api.command.RawCommand;
-import me.aurium.beetle.core.command.CommandData;
 import net.kyori.adventure.text.Component;
 
-public class VelocityCommandData implements CommandData {
+public class VelocityCommandData {
```

Comment:
Does this class do anything now that these changes have been applied?

Reply:
Nope, and it's likely to be torn out in the command-removing update (all command functionality is being moved over to branch)